src/localization/english.ts

```diff
@@ -37,14 +37,14 @@ export var englishStrings = {
   numericMax: "The '{0}' should be equal or less than {1}",
   invalidEmail: "Please enter a valid e-mail address.",
   invalidExpression: "The expression: {0} should return 'true'.",
-  urlRequestError: "The request returned error '{0}'. {1}",
+  urlRequestError: "A network service has returned the following error '{0}'. {1}",
```

Comment:
- 'A network service' is quite vague. Which network service'? What causes the error?
- I would change 'has returned' to 'returned' since you are referring to the past tense.

Reply:
This error is displayed to users who probably have no knowledge of surveyjs, computer and network stuff. An error message that appears with "the request returned error ..." does not tell the user what is wrong. You are right, there is no context that loading answers from external resources failed.

This message should give the user an indication of what is going wrong so that the user can send a helpful bug report to the survey creator.